### PR TITLE
Update execution-guide.md

### PR DESCRIPTION
**Pull request by pkchau-skylight**
- added 'prep' section with link to implementation guide
- Step 11: added in "If you see multiple piplines with this name, select the one at the top (the most recently opened one)."
- Step 14: added in parentheses to select the pencil icon 
- Added Step 16: "Select the bash option."

### CODE REVIEW

File: docs/execution-guide.md [hunk 1]
```diff
@@ -1,5 +1,15 @@
 # Execution Guide: How to run data through the Azure Starter Kit
 
+### Set up accounts prior to running data though the pipeline
+First, please confirm that you meet the following user requirements:
+1. Owner access to the Azure subscription where you would like to deploy the PHDI Starter Kit.
+1. A GitHub account with a verified email address.
+1. Access to create new repositories in the GitHub account or organization where your copy of this repository will be created.
+1. If you are planning to use an organization account, you must be able to authorize the GitHub CLI to interact with the organization.
```
Comment (DanPaseltiner) [inline docs/execution-guide.md:8]: The requirements for running the Execution Guide are more minimal than the Implementation Guide.

```suggestion
First, please confirm that you meet the following user requirements:
1. Access to an Azure resource group where the starter kit has been deployed. Access must include the following roles:
    - `Storage Blob Data Contributor` on the PHI storage account
    - `FHIR Data Contributor` on the FHIR server
2. All of the files in the [sample-data/](../sample-data/) directory easily available on your computer.
```

File: docs/execution-guide.md [hunk 2]
```diff
@@ -1,5 +1,15 @@
...
+
+If you have not implemented the pipeline or do not meet one of the user requirements above, please follow the steps in this [Implementation Guide](https://github.com/CDCgov/phdi-azure/blob/main/docs/implementation-guide.md).
```
Comment (DanPaseltiner) [inline docs/execution-guide.md:10]:
```suggestion


If you have not implemented the pipeline, please follow the steps in this [Implementation Guide](https://github.com/CDCgov/phdi-azure/blob/main/docs/implementation-guide.md). 

If you or your organization, have deployed the starter kit, but you do not have sufficient access please contact your organization's Azure administrator.

```

File: docs/execution-guide.md [hunk 3]
```diff
@@ -29,11 +39,12 @@ If you would like, feel free to confirm that this is the case by inspecting the
...
 1. You should now see a diagram showing the steps of the pipeline. To view the status of our attempted run, click `Monitor` on the left sidebar.![azure-pipeline-select-monitor](./images/azure-pipeline-select-monitor.png)
 1. We should now see that the ingestion pipeline has processed one message successfully.![azure-ingestion-single-execution](./images/azure-ingestion-single-execution.png)
-1. To view the JSON configuration for the pipeline and a visualization of the process, go to the `AUTHOR` tab.![azure-workflow-source](./images/azure-workflow-source.png)
+1. To view the JSON configuration for the pipeline and a visualization of the process, go to the `AUTHOR` tab (pencil icon).![azure-workflow-source](./images/azure-workflow-source.png)
 1. To view the cleaned and enriched data in the FHIR server, visit https://shell.azure.com to open the Cloud Shell.![azure-cloud-shell](./images/azure-cloud-shell.png)
+1. Select the bash option.
```
Comment (DanPaseltiner) [inline docs/execution-guide.md:47]: 🚀

**Review by DanPaseltiner — COMMENTED** [review]
Thanks for putting this together @pkchau-skylight! I think we need to tweak the user requirements a bit (see suggestions below).

File: docs/execution-guide.md [hunk 4]
```diff
@@ -29,11 +39,12 @@ If you would like, feel free to confirm that this is the case by inspecting the
 1. To see that the pipeline has executed, search for `Data factories` and go to the Data factories page.![azure-search-data-factories](./images/azure-search-data-factories.png)
 1. Select your data factory, which will be titled `phdi-{environment}-data-factory`.![azure-select-ingestion-pipeline](./images/azure-select-ingestion-pipeline.png)
 1. Launch the Data Factory Studio by clicking the blue button that says `Launch studio`.![azure-data-factory-launch-studio](./images/azure-data-factory-launch-studio.png)
-1. Select your ingestion pipeline, which will be titled `phdi-{environment}-ingestion`.![azure-select-ingestion-pipeline-in-studio](./images/azure-select-ingestion-pipeline-in-studio.png)
+1. Select your ingestion pipeline, which will be titled `phdi-{environment}-ingestion`. If you see multiple piplines with this name, select the one at the top (the most recently opened one).![azure-select-ingestion-pipeline-in-studio](./images/azure-select-ingestion-pipeline-in-studio.png)
```
Comment (nickclyde) [inline docs/execution-guide.md:42]: I think we noticed that the pipeline won't be listed if it's your first time opening data factory. So we should add a disclaimer about this and maybe just say to go straight to the Monitor tab if there's no pipelines listed?

**Review by nickclyde — COMMENTED** [review]
Looks good, just one small comment!

**Review by nickclyde — APPROVED** [review]
This looks great, thank you so much for adding these changes Peggy! 

Just a heads up FYSA: In markdown, you can do a numbered list all starting with `1.` instead of incrementing, and when it renders it will show 1,2,3, etc. That way if you ever need to shift steps around or remove steps, you don't have to change the numbering of following steps!

File: docs/execution-guide.md (hunk 4 above)
Comment (pkchau-skylight) [reply docs/execution-guide.md]: Sg! I'll add the disclaimer for now and then might ask you a follow up Q about this later on whether it's best to just flip the steps regardless!

**Comment by pkchau-skylight** [comment]
@nickclyde @DanPaseltiner When I add in a new line, it auto increments the numbers in the following steps--is there a way to not have it do this?

**Comment by DanPaseltiner** [comment]
@pkchau-skylight can you point me to where this is happening in the doc?

**Pull request merged**